.github/workflows/build.yml

```diff
@@ -206,57 +209,36 @@ jobs:
         run: rm c_sharp/src/org/ldk/enums/*.cs c_sharp/src/org/ldk/impl/*.cs c_sharp/src/org/ldk/structs/*.cs
       - name: Build Windows C# Bindings
         run: |
-          export LDK_GARBAGECOLLECTED_GIT_OVERRIDE="$(git describe --tag HEAD)"
+          # run dotnet fsi first time so that welcome message is not printed when it's invoked again
+          dotnet fsi nugetPreRelease.fsx $NUGET_PACKAGE_VERSION
```

Comment:
@webwarrior-ws this is too ugly, just use DOTNET_NOLOGO as per the dotnet documentation; also, do not call `dotnet fsi nugetPreRelease.fsx` several times or you might get different versions each time, take in account that the result is time-sensitive! e.g. if you call it 1 minute later it will give you a different result!!!!